R/bin_time.R

```diff
@@ -154,12 +154,12 @@ bin_time <- function(occdf, bins, method = "mid", reps = 100) {
 
     # Generate empty column for recording the number of bins an occurrence
     # appears in, and empty columns for the new bin allocation and midpoint.
-    occdf$n_bins <- NA
+    occdf$n_bin <- NA
```

Comment:
This can be kept as n_bins (since each individual value in the column refers to the number of bin**s**)